src/jit/hwintrinsicArm64.cpp

```diff
@@ -213,6 +169,21 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
             break;
     }
 
+    // Simd instantiation type check
+    if (simdClass != nullptr)
+    {
+        simdBaseType = getBaseTypeAndSizeOfSIMDType(simdClass, &simdSizeBytes);
+
+        if (simdBaseType == TYP_UNKNOWN)
+        {
+            // TODO-FIXME Add CORINFO_HELP_THROW_TYPE_NOT_SUPPORTED
```

Comment:
Nit: Please follow the conventions for `TODO` comments (https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#715-todo) - e.g. I think this should probably be `TODO-Bug`

Reply:
👍 I'll fix this in #16085 (where this stray bit of code came from)

Reply:
Sounds good. Thanks